turbinia/processors/mount_local.py

```diff
@@ -34,6 +34,20 @@
     'Total number of loop devices failed to detach')
 
 
+def IsLosetup(path):
+  # Check that the device was attached
+  losetup_cmd = ['sudo', 'losetup', '-a']
+  log.info('Running: {0:s}'.format(' '.join(losetup_cmd)))
+  try:
+    output = subprocess.check_output(losetup_cmd)
+  except subprocess.CalledProcessError as e:
+    raise TurbiniaException(
+        'Could not check losetup device status {0!s}'.format(e))
+  if output.find(path.encode('utf-8')) == -1:
```

Comment:
I think we might need to add some anchors here (not sure if that means switching to using the `re` module or if there is a better way to do that with `find`) because it looks to me like this could erroneously return true when checking for `/dev/loop1` and `/dev/loop10` exists.

Reply:
Hmm I tested through a debugger `output` having loop10 in string 
```
 b'/dev/loop10: [0063]:3026494 (/workspaces/turbinia/test_data/ext2.raw)\n/dev/loop0: [0063]:3026494 (/workspaces/turbinia/test_data/ext2.raw)\n'
```
and path being `/dev/loop1` as well as switching `output` to be loop1 and path being loop10, returning False in both cases/WAI so I think we're covered in this case and don't need to adjust.

Reply:
Hmm I tested through a debugger `output` having loop10 in string 
```
 b'/dev/loop10: [0063]:3026494 (/workspaces/turbinia/test_data/ext2.raw)\n/dev/loop0: [0063]:3026494 (/workspaces/turbinia/test_data/ext2.raw)\n'
```
and path being `/dev/loop1` as well as switching `output` to be loop1 and path being loop10, returning False in both cases/WAI so I think we may actually be covered in this case